### ISSUE TITLE
Trying to use the RN code for CLEVR dataset which results in overfitting.

### ISSUE THREAD

**Issue by saharudra** [vmichals/FigureQA-baseline#1]
Hi,

Not sure if this is the right place to ask this question, but I adapted the RN code to train on the CLEVR dataset. I set up the hyperparameters by following the paper but I am facing the issue of overfitting. The validation accuracy gets stuck at 50%.

Any suggestions on this will be helpful.
